include/nbl/builtin/glsl/ext/RadixSort/default_histogram.comp

```diff
@@ -99,24 +103,38 @@ void nbl_glsl_ext_RadixSort_setHistogram(in uint idx, in uint val)
 #define _NBL_GLSL_EXT_RADIXSORT_HISTOGRAM_SET_HISTOGRAM_DEFINED_
 #endif
 
+#ifndef _NBL_GLSL_SCRATCH_SHARED_SIZE_DEFINED_
+#define _NBL_GLSL_SCRATCH_SHARED_SIZE_DEFINED_ (_NBL_GLSL_WORKGROUP_SIZE_ * 2)
+shared uint scratch_shared[_NBL_GLSL_SCRATCH_SHARED_SIZE_DEFINED_];
+#endif
+#define _NBL_GLSL_SCRATCH_SHARED_DEFINED_ scratch_shared
```

Comment:
do not ever define this macro unconditionally by yourself (move this definition into the `#ifndef _NBL_GLSL_SCRATCH_SHARED_SIZE_DEFINED_` clause), just use `_NBL_GLSL_SCRATCH_SHARED_DEFINED_` instead of `scratch_shared` throughout the code